indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -299,9 +316,13 @@ protected void innerMap(
       // and they contain the columns as they show up in the segment after ingestion, not what you would see in raw
       // data
       byte[] serializedInputRow = inputRow instanceof SegmentInputRow ?
-                                  InputRowSerde.toBytes(inputRow, combiningAggs, reportParseExceptions)
+                                  InputRowSerde.toBytes(typeHelperMap, inputRow, combiningAggs, reportParseExceptions)
                                                                       :
-                                  InputRowSerde.toBytes(inputRow, aggregators, reportParseExceptions);
+                                  InputRowSerde.toBytes(typeHelperMap, inputRow, aggregators, reportParseExceptions);
+
+      if (serializedInputRow == null) {
+        return;
```

Comment:
This should increment INVALID_ROW_COUNTER and log a debug message. However, this comment may be moot if you take the suggestion to skip just the column and not the entire row.

Reply:
Added a counter increment and debug message for now